package.json

```diff
@@ -74,6 +74,7 @@
     "stylelint-prettier": "^1.2.0",
     "stylelint-scss": "^3.21.0",
     "vuepress": "^1.8.2",
+    "vuepress-plugin-demo-code": "^0.6.0",
```

Comment:
@detiwari003 Is this dependency required ?. Where is it being used in the docs site ?

Reply:
Fixed